tools/check/check_code_quality.sh

```diff
@@ -67,6 +67,9 @@ echo "Search for forbidden patterns in code..."
 ${searchForbiddenStringsScript} ./tools/check/forbidden_strings_in_code.txt \
     ./matrix-sdk-android/src/main/java \
     ./matrix-sdk-android-flow/src/main/java \
+    ./library/core-utils/src/main/java \
```

Comment:
I don't know the internals of `searchForbiddenStringsScript` but perhaps we could glob this to automatically pickup source directories? `**/src/main/java` (it looks like we want to check all the modules~)

Reply:
yes, it's a pain to add all the modules one by one...